packages/atlas/atlas.config.yml

```diff
@@ -36,6 +36,12 @@ features:
     googleConsoleClientId: '$VITE_GOOGLE_CONSOLE_CLIENT_ID'
     youtubeSyncApiUrl: '$VITE_YOUTUBE_SYNC_API_URL'
     youtubeCollaboratorMemberId: '$VITE_YOUTUBE_COLLABORATOR_MEMBER_ID'
+    tiersDefinition:
+      tiers:
+        - subscribers: 2
```

Comment:
maybe `minimumSubscribers`?

and maybe let's add small comment next to `tiers` to explain what this is

Reply:
let's also add `multiplier` field to each tier